src/backend/marsha/page/api.py

```diff
@@ -0,0 +1,44 @@
+"""Declare API endpoints with Django RestFramework viewsets."""
+import django_filters
+from rest_framework import filters, mixins, viewsets
+
+from . import serializers
+from .models import Page
+
+
+class PageViewSet(
+    mixins.ListModelMixin,
+    mixins.RetrieveModelMixin,
+    viewsets.GenericViewSet,
+):
+    """Viewset for the API of the ``page`` object."""
+
+    queryset = Page.objects.filter(is_published=True).all()
+    serializer_class = serializers.PageSerializer
+    filter_backends = [
+        filters.OrderingFilter,
+        django_filters.rest_framework.DjangoFilterBackend,
+    ]
+    ordering_fields = ["position", "created_on"]
+    ordering = ["position", "-created_on"]
+    lookup_field = "slug"
+
+    permission_classes = []
```

Comment:
Why not using the permission `rest_framework.permissions.AllowAny` ? And doing this the `get_permissions` seems useless. WDYT ?

Reply:
True